<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApertureLayer</class>
 <widget class="QWidget" name="ApertureLayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>1280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>1280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="ring_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>105</y>
     <width>800</width>
     <height>1014</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>1014</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>1014</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/1x/Property_1_green.png);</string>
   </property>
   <widget class="QWidget" name="info_widget" native="true">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>869</y>
      <width>570</width>
      <height>128</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>570</width>
      <height>128</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>570</width>
      <height>128</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#info_widget
{
	background-image: url(:/images/1x/mainInfo_green.png);
}</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="leftMargin">
      <number>48</number>
     </property>
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>48</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>24</number>
       </property>
       <property name="leftMargin">
        <number>40</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>40</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item>
        <widget class="QLabel" name="label_head_photo">
         <property name="minimumSize">
          <size>
           <width>88</width>
           <height>88</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>88</width>
           <height>88</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resource.qrc">:/images/1x/PersonInfoNoPhoto.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_name">
         <property name="minimumSize">
          <size>
           <width>102</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgba(255, 255, 255, 1);
font-family: Noto Sans SC;
font-size: 40px;</string>
         </property>
         <property name="text">
          <string>李*六</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="temperature_widget" native="true">
       <property name="minimumSize">
        <size>
         <width>172</width>
         <height>64</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>172</width>
         <height>64</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 32px;
background: rgba(255, 255, 255, 1);</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>8</number>
        </property>
        <property name="leftMargin">
         <number>11</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="label_temperature_icon">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resource.qrc">:/images/1x/temperature_green.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_temperature_val">
          <property name="styleSheet">
           <string notr="true">color: rgba(39, 149, 15, 1);
font-family: Noto Sans SC;
font-weight: bold;
font-size: 32px;</string>
          </property>
          <property name="text">
           <string>36.9℃</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_tip">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>1121</y>
     <width>540</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>540</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>540</width>
     <height>90</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(255, 255, 255, 1);
font-family: Noto Sans SC;
font-size: 40px;
letter-spacing: 4%;
border-radius: 45px;
border: 1px solid rgba(232, 232, 232, 0.5);
box-sizing: border-box;
background: rgba(5, 25, 45, 0.65);
backdrop-filter: blur(20px);</string>
   </property>
   <property name="text">
    <string>请靠近屏幕</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
